verticapy/plotting/_plotly/density.py

```diff
@@ -40,7 +40,7 @@ def _init_style(self) -> None:
             "width": 700,
             "height": 500,
             "autosize": False,
-            "xaxis_title": self.layout["x_label"],
+            "xaxis_title": self.layout["x_label"][1:-1],
```

Comment:
You should never use this kind of logic directly in the code. Imagine that the user passes a customised layout, you'll always retrieve the first and the last character.

Reply:
Got it. Thanks for pointing out. 
Updated it.